### PR TITLE
#10-1 new-profile_edit(PR#6)

### PR DESCRIPTION
**Pull request by kokioba95**
## Description / Purpose (checkpoints)
It is confirmation of this PR, profile_edit function & layout,.

Fixes issue: 
<https://docs.google.com/spreadsheets/d/1Ear__g58irVlaI_tpzWUkf__JRcBz8u2d2DdRYKMiDk/edit?usp=sharing> <Back Log #10>

## How To Test This PR
1. Go login page(Ex.http://localhost:8086/login)
2. Input User Credentials
3. Click Login 
4. my_profile page
5. Press the "Edit profile" button
6. Enter all content
7. Press the　”Update”button
8. (Return to my_profile page)
 
 **Commands**
　php artisan make:request EditProfile
　php artisan make:controller UserController

 **Screenshots**
 - < https://drive.google.com/drive/folders/1UX_at7VJu0OeH5Mb7Eo7ULK3hSw7Ws-W?usp=sharing >

### CODE REVIEW

File: app/Http/Controllers/UserController.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+use Illuminate\Support\Facades\Hash;
+
+class UserController extends Controller
+{
+    public function edit($id)
+    {
+        $user = User::find($id);
+        return view('profile_edit', compact('user'));
+    }
+    public function update(EditProfile $request, $id)
```
Comment (AubreyV) [inline app/Http/Controllers/UserController.php]: Please put a new line in between functions. It's helps in making your code readable.

File: app/Http/Controllers/UserController.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Http\Controllers;
+use Illuminate\Http\Request;
```
Comment (AubreyV) [inline app/Http/Controllers/UserController.php]: Put a space after namespace. It's important to separate unrelated code so that it is easy to read.

File: resources/views/my_profile.blade.php [hunk 3]
```diff
@@ -46,5 +46,9 @@
 
 
 </div>
+<<<<<<< Updated upstream
```
Comment (AubreyV) [inline resources/views/my_profile.blade.php]: Do not push git conflicts.

File: resources/views/my_profile.blade.php [hunk 4]
```diff
@@ -46,5 +46,9 @@
...
 @endsection
 
+=======
```
Comment (AubreyV) [inline resources/views/my_profile.blade.php]: Do not push git conflicts.

File: resources/views/my_profile.blade.php [hunk 5]
```diff
@@ -46,5 +46,9 @@
...
+@endsection
+>>>>>>> Stashed changes
```
Comment (AubreyV) [inline resources/views/my_profile.blade.php]: Do not push git conflicts.

**Comment by hayato0309** [comment]
After clicking "Edit Profile" button in /my_profile page, 404 is displayed.

File: app/Http/Controllers/UserController.php (hunk 1 above)
Comment (kokioba95) [reply app/Http/Controllers/UserController.php]: I edited it.

File: app/Http/Controllers/UserController.php (hunk 2 above)
Comment (kokioba95) [reply app/Http/Controllers/UserController.php]: I edited it.

File: resources/views/my_profile.blade.php (hunk 3 above)
Comment (kokioba95) [reply resources/views/my_profile.blade.php]: I deleted it.

File: resources/views/my_profile.blade.php (hunk 4 above)
Comment (kokioba95) [reply resources/views/my_profile.blade.php]: I deleted it.

File: resources/views/my_profile.blade.php (hunk 5 above)
Comment (kokioba95) [reply resources/views/my_profile.blade.php]: I deleted it.
